detection_rules/utils.py

```diff
@@ -87,18 +87,18 @@ def get_json_iter(f):
 
 def get_path(*paths) -> str:
     """Get a file by relative path."""
-    return os.path.join(ROOT_DIR, *paths)
+    return str(ROOT_DIR.joinpath(*paths))
```

Comment:
nit: Not to scope creep but in a lot of places, we convert immediately to `Path` anyway. Does it make sense to just convert the return type?

Reply:
+1, `*_path` suffix also suggests that it  will be a `Path` obj

Reply:
Yea, I agree. Will need more testing for this approach, but it is a better one and can reduce complexity for some of our conversions. :+1: